diff -Naur dhewm3-1.5.0.old/neo/CMakeLists.txt dhewm3-1.5.0/neo/CMakeLists.txt
--- dhewm3-1.5.0.old/neo/CMakeLists.txt	2019-01-06 16:58:20.269094586 +0100
+++ dhewm3-1.5.0/neo/CMakeLists.txt	2019-01-06 17:02:03.725252014 +0100
@@ -52,6 +52,7 @@
 option(DEDICATED	"Build the dedicated server" OFF)
 option(ONATIVE		"Optimize for the host CPU" OFF)
 option(SDL2			"Use SDL2 instead of SDL1.2" ON)
+option(ZFAIL		"Use Carmack's reverse" OFF)
 
 if(NOT CMAKE_SYSTEM_PROCESSOR)
 	message(FATAL_ERROR "No target CPU architecture set")
@@ -766,6 +767,9 @@
 		sys/win32/win_syscon.cpp
 		sys/win32/SDL_win32_main.c
 	)
+	if(ZFAIL)
+		add_definitions( -DDEFY_PATENTS )
+	endif()
 
 	set(src_sys_core
 		sys/glimp.cpp
diff -Naur dhewm3-1.5.0.old/neo/renderer/draw_common.cpp dhewm3-1.5.0/neo/renderer/draw_common.cpp
--- dhewm3-1.5.0.old/neo/renderer/draw_common.cpp	2019-01-06 16:58:20.324095608 +0100
+++ dhewm3-1.5.0/neo/renderer/draw_common.cpp	2019-01-06 17:02:03.726252033 +0100
@@ -1140,6 +1140,7 @@
 		return;
 	}
 
+#ifndef DEFY_PATENTS
 	// patent-free work around
 	if ( !external ) {
 		// "preload" the stencil buffer with the number of volumes
@@ -1151,6 +1152,17 @@
 		GL_Cull( CT_BACK_SIDED );
 		RB_DrawShadowElementsWithCounters( tri, numIndexes );
 	}
+#else
+	// Z-fail, uses stuff described in US6384822 held by Creative Labs in the United States
+	if ( !external ) {
+		qglStencilOp( GL_KEEP, tr.stencilDecr, GL_KEEP );
+		GL_Cull( CT_FRONT_SIDED );
+		RB_DrawShadowElementsWithCounters( tri, numIndexes );
+		qglStencilOp( GL_KEEP, tr.stencilIncr, GL_KEEP );
+		GL_Cull( CT_BACK_SIDED );
+		RB_DrawShadowElementsWithCounters( tri, numIndexes );
+	}
+#endif
 
 	// traditional depth-pass stencil shadows
 	qglStencilOp( GL_KEEP, GL_KEEP, tr.stencilIncr );
